Exiled.API/Features/Player.cs

```diff
@@ -1016,21 +1071,21 @@ public bool IsSpawnProtected
         /// </summary>
         /// <param name="side">The players' side.</param>
         /// <returns>The filtered <see cref="IEnumerable{T}"/>.</returns>
-        public static IEnumerable<Player> Get(Side side) => List.Where(player => player.Role.Side == side);
+        public static IEnumerable<Player> Get(Side side) => InstantiatedSideToPlayers?[side];
 
         /// <summary>
         /// Gets a <see cref="Player"/> <see cref="IEnumerable{T}"/> filtered by team. Can be empty.
         /// </summary>
         /// <param name="team">The players' team.</param>
         /// <returns>The filtered <see cref="IEnumerable{T}"/>.</returns>
-        public static IEnumerable<Player> Get(Team team) => List.Where(player => player.Role.Team == team);
+        public static IEnumerable<Player> Get(Team team) => InstantiatedTeamToPlayers?[team];
 
         /// <summary>
         /// Gets a <see cref="Player"/> <see cref="IEnumerable{T}"/> filtered by role. Can be empty.
         /// </summary>
         /// <param name="role">The players' role.</param>
         /// <returns>The filtered <see cref="IEnumerable{T}"/>.</returns>
-        public static IEnumerable<Player> Get(RoleType role) => List.Where(player => player.Role == role);
+        public static IEnumerable<Player> Get(RoleType role) => InstantiatedRolesToPlayers?[role];
```

Comment:
Why you use "?" It's should not being null. if it's null it's because the server have not finish to start and the dev have done a skillissue because calling Player.Get before server have finish to start is pretty useless

Reply:
Fine.. :kek